### PR TITLE
Update signup

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?181122

Old account creation page:
screen shot 2015-09-09 at 6 12 34 pm

New account creation page:
screen shot 2015-09-09 at 6 12 02 pm

Old invitation acceptance page:
screen shot 2015-09-09 at 6 16 13 pm

New invitation acceptance page:
screen shot 2015-09-09 at 6 22 11 pm

Asking for quick sanity QA, as a bug in account creation would be hard for end users to report.

cc @dimagi/product 

@emord / @kaapstorm

### CODE REVIEW

File: corehq/apps/registration/forms.py [hunk 1]
```diff
@@ -69,16 +66,8 @@ class NewWebUserRegistrationForm(DomainRegistrationForm):
...
-            self.fields['hr_name'].required = False
-            if kwargs.get('initial', {}).get('hr_name'):
-                self.fields['hr_name'].widget = BootstrapDisabledInput(attrs={'class': 'input-xlarge'})
-            else:
-                self.fields['hr_name'].widget = forms.HiddenInput()
-
-    def clean_full_name(self):
-        data = self.cleaned_data['full_name'].split()
-        return [data.pop(0)] + [' '.join(data)]
+        if not kwargs.get('initial', {}).get('create_domain', True) or self.data.get('create_domain', True) == "False":
```
Comment (dimagimon) [inline corehq/apps/registration/forms.py]: E501 line too long (119 > 115 characters)

**Comment by czue** [comment]
nice!

File: corehq/apps/registration/forms.py [hunk 2]
```diff
@@ -46,9 +46,6 @@ class NewWebUserRegistrationForm(DomainRegistrationForm):
     """
     Form for a brand new user, before they've created a domain or done anything on CommCare HQ.
     """
-    full_name = forms.CharField(label=_('Full Name'),
```
Comment (benrudolph) [inline corehq/apps/registration/forms.py]: do we use the full_name in lots of places? just want to make sure that none of our emails only use full_name (without fallback) and now new users won't have one

**Comment by benrudolph** [comment]
also a fan of the quick project setup, cool idea!

File: corehq/apps/registration/forms.py (hunk 2 above)
Comment (orangejenny) [inline corehq/apps/registration/forms.py]: good point, I'll take a scan of the places `full_name` is used

Comment (emord) [inline corehq/apps/registration/forms.py]: Ha. I recently used `full_name` for interrupt https://github.com/dimagi/commcare-hq/pull/8109/files

Just realizing now that keeping the username kwarg was probably not the most clear idea

Comment (amsagoff) [inline corehq/apps/registration/forms.py]: hey - this is now stalling on some objections from marketing - we may need
to put firstname/lastname back in there. Will update you when this thread
is resolved (hopefully soon)

Comment (amsagoff) [inline corehq/apps/registration/forms.py]: Update: we need to put first name and last name back in there, as optional
fields. Jenny, happy to give context over voice, but basically hubspot has
some compelling evidence that having names helps significantly with
marketing campaigns.

**Comment by orangejenny** [comment]
reinstated full name, should be good to go

File: corehq/apps/registration/forms.py (hunk 2 above)
Comment (orangejenny) [inline corehq/apps/registration/forms.py]: Put full name back. How strongly do you feel that it should be optional? As long as the input is there, I feel like we might as well leave it required, unless there's some set of users who are opposed to entering their names...?

**Comment by orangejenny** [comment]
nm, wait on answer from Amelia re: requiring name before merging

File: corehq/apps/registration/forms.py (hunk 2 above)
Comment (orangejenny) [inline corehq/apps/registration/forms.py]: @dimagi/product ping, should name should be optional or required?

Comment (amsagoff) [inline corehq/apps/registration/forms.py]: Good to go on required now, for some reason I thought it was optional on
prod currently, but just tested and it's not.

Could be fun to A/B test optional vs. required, but comms through hubspot
can't handle that.

Thanks for pinging

Comment (orangejenny) [inline corehq/apps/registration/forms.py]: sounds good

**Comment by orangejenny** [comment]
ready to merge for real @emord

**Pull request merged**